api/v1beta1/types.go

```diff
@@ -325,11 +333,12 @@ var (
 
 	DefaultClusterName string = "cluster-api"
 
+	DefaultVmName             string = "cluster-api-vm"
 	DefaultVmSubregionName    string = "eu-west-2a"
-	DefaultVmImageId          string = "ami-2fe74243"
+	DefaultVmImageId          string = "ami-e1a786f1"
```

Comment:
Is this OMI a duplicate from official one so we don't change it regularly ?

Reply:
create from image builder and github action.
It will be change from omi name to image name.